core/primitives/src/version.rs

```diff
@@ -88,11 +91,11 @@ pub enum ProtocolFeature {
 
 /// Current latest stable version of the protocol.
 #[cfg(not(feature = "nightly_protocol"))]
-pub const PROTOCOL_VERSION: ProtocolVersion = 41;
+pub const PROTOCOL_VERSION: ProtocolVersion = 48;
```

Comment:
this should be updated to 42. Stable protocol version is not in sync with nightly protocol version

Reply:
Fixed